app/core/advanced_cache.py

```
@@ -0,0 +1,227 @@
+                reverse=True
+            )
+            
+            # Warm most frequently accessed keys
+            for key, _ in sorted_keys[:100]:  # Warm top 100 items
+                try:
+                    value = loader_func(key)
+                    if value is not None:
+                        self.cache.put(key, value)
+                        if hasattr(self.cache, 'stats'):
+                            self.cache.stats.warm_loads += 1
+                except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)